src/index.css

```
@@ -503,47 +604,41 @@ section {
-  62.5% {
-    transform: translateY(-62.5%);
-  }
-  75% {
-    transform: translateY(-75%);
-  }
-  87.5% {
-    transform: translateY(-87.5%);
-  }
+/* 4 unique words + a repeat of the first; the final stop equals the start,
+   so the snap-back is invisible. */
+@keyframes wordRotate {
```

Stylelint (17.12.0)

[error] Expected keyframe name "wordRotate" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)